src/app/[locale]/products/page.tsx

```
@@ -1,9 +1,15 @@
 import React from 'react'
+import FilterSidebar from '@components/features/filters-sidebar/filters-sidebar'
 
```

ESLint

[error] Missing semicolon.

(semi)